### DISCUSSION TITLE
Is Ubuntu 24.04 slower than 22.04 with ffmpeg/nvenc in WSL2?  Drop of 100-150fps encoder time.

### DISCUSSION THREAD

**Discussion by IronWolve** [microsoft/WSL#11536] [General]
Just noticed, it was a bit slower in latest 24.04, I still have 22.04 installed.  Using repo ffmpeg, just wondering if its just me.

**Comment by maxborowsky** [comment]
@IronWolve, not only do you have problems with slow Ubuntu 24.04, this has been noticed for both hardware and virtual machines. Here are some links:

https://discourse.ubuntu.com/t/24-04-considerably-slower-than-20-04-or-22-04-for-some-high-system-percentage-usage-cases/41987
https://askubuntu.com/questions/1511908/why-is-ubuntu-desktop-24-04-significantly-slower-on-raspberry-pi-5-than-ubuntu-2

It seems to me that 24.04 is completely unprepared for everyday use.
